platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -168,21 +173,17 @@ a {
 table {
   background: $body_background;
   width: 100%;
+  font-size: 13px;
+  line-height: 23px;
   border-collapse: collapse;
   border-spacing: 0;
   overflow: auto;
   margin: 0 0 0.85em;
 }
 
-tr {
-  &:nth-child(2n) {
-    background-color: $table_alt_row_color;
-  }
-}
-
 th, td {
   padding: 6px 13px;
-  border: 1px solid $table_border_color;
+  border: 1px solid $keyline_color;
```

Comment:
Since this documentation is most likely to be viewed on Retina displays, we can also use darker, half-pixel-wide borders to achieve a crisper look without sacrificing subtlety.